### PR TITLE
[Core][Triangle3D3] correcting projection check

### PR DESCRIPTION
**Pull request by philbucher**
I suddenly had many of those warnings even though I project the points before checking for `IsInside`
I noticed that you project along the direction PointToProject-Center which is not correct, you should project along the Normal of the Triangle

Here is a screenshot of an example:
Screenshot from 2019-05-08 15-25-03

the point with name "Vicentes_projection" should be in the same location as the point "projected_point" since "projected_point" is what I give into the `IsInside` and it already is in the plane of the triangle

I hope I explained myself not too bad

### CODE REVIEW

**Comment by loumalouomega** [comment]
I cannot check the code right now, but I think I use the normal...

File: kratos/geometries/triangle_3d_3.h [hunk 1]
```diff
@@ -875,24 +875,19 @@ template<class TPointType> class Triangle3D3
...
-
         // We compute the normal to check the normal distances between the point and the triangles, so we can discard that is on the triangle
-        const CoordinatesArrayType aux_point = ZeroVector(3);
-        const array_1d<double, 3> normal = this->UnitNormal(aux_point);
+        const array_1d<double, 3> normal = this->UnitNormal(this->Center());
 
         // We compute the distance, if it is not in the pane we
         const Point point_to_project(rPoint);
         Point point_projected;
-        const array_1d<double,3> vector_points = rPoint - r_center.Coordinates();
```
Comment (philbucher) [inline kratos/geometries/triangle_3d_3.h:888]: This is the direction you used and its not the normal 

I corrected it, now it does use the normal

**Comment by loumalouomega** [comment]
Please wait, I will check ASAP.

**Comment by philbucher** [comment]
Sure, no hurry

File: kratos/geometries/triangle_3d_3.h (hunk 1 above)
Comment (loumalouomega) [reply kratos/geometries/triangle_3d_3.h:888]: OK I see, I used the wrong utility. Wait a second.

Besides the algorithm is quite common, here an example:

https://stackoverflow.com/questions/9605556/how-to-project-a-point-onto-a-plane-in-3d

Comment (loumalouomega) [reply kratos/geometries/triangle_3d_3.h:888]: I will use instead FastProject, you are computing twice the normal (performance)

**Review by loumalouomega — APPROVED** [review]
Approving now

File: kratos/geometries/triangle_3d_3.h (hunk 1 above)
Comment (philbucher) [reply kratos/geometries/triangle_3d_3.h:888]: makes sense, thx
will test it before merging

**Comment by philbucher** [comment]
works fine, thx

**Pull request merged**